### DISCUSSION TITLE
Docker build for project on Mac

### DISCUSSION THREAD

**Discussion by akshayjain3450** [rilldata/rill#4349] [Q&A]
Does anyone has ever build this project using Docker on Mac? What changes did they make to their Dockerfile and how have you done that?

Looking for some help on this. @AndrewRTsao

**Comment by AndrewRTsao** [comment]
Docker is not an officially supported install path for Rill. If you are looking to install Rill on your local machine, especially with something like MacOS, we would recommend using the default installer (i.e. `rill.sh`). In case you're curious of the contents of the installer and how it works, you can download and inspect it locally, e.g.:
```bash
wget -O rill.sh https://rill.sh
```

One thing to note is that we will actually be expanding this install script in the near future to support additional options, such as a non-sudo required installation path. If using Rill with Docker is a hard requirement, we'd love to learn more about your specific use case / requirements and see if there might be other ways that we can help out.
